README.rst

```diff
@@ -64,15 +70,28 @@ key to download a request using Zyte API. Full list of parameters is provided in
 
 .. code-block:: python
 
-    yield scrapy.Request(
-        "http://books.toscrape.com/",
-        callback=self.parse,
-        meta={
-            "zyte_api": {
-                "browserHtml": True,
-                "geolocation": "US",
-                "javascript": True,
-                "echoData": {"something": True}
-            }
-        }
-    )
+   import scrapy
+
+   class TestSpider(scrapy.Spider):
+       name = 'test'
+       start_urls = ['http://books.toscrape.com/']
+
+       def start_requests(self):
+
+               yield scrapy.Request(url="http://books.toscrape.com/", callback=self.parse,
+                                    meta={
+                                        "zyte_api": {
+                                            "browserHtml": True,
+                                            # You can set any GEOLocation region you want.
+                                            "geolocation": "US",
+                                            "javascript": True,
+                                            "echoData": {"something": True}
+                                        }
+                                    })
+
+       def parse(self, response, **kwargs):
+           yield{
+               'URL': response.url,
+               'status': response.status,
+               'HTML': response.body
+           }
```

Comment:
Let's format this code according to [black's ](https://github.com/psf/black) standards to fix the indentions and spacings.

Reply:
Yes, used the black's standard for formatting now.

![image](https://user-images.githubusercontent.com/74107530/152358932-7223ae37-27ab-4df7-bd60-1aed2a848c5b.png)